keel-plugin/src/main/kotlin/com/netflix/spinnaker/keel/plugin/TaskLauncher.kt

```diff
@@ -96,11 +96,11 @@ class TaskLauncher(
         Task(id = it.taskId, name = description)
       }
 
-  private val Resource<*>.notifications: List<EchoNotification>
+  private val Resource<*>.notifications: List<OrcaNotification>
     get() = deliveryConfigRepository
       .environmentFor(id)
-      .notifications
-      .map { it.toEchoNotification() }
+      ?.notifications
```

Comment:
why is this `?` now? I don't think that environment for `id` should ever be null - resources must be in an environment. Is the signature wrong on that method? (i'm wondering if I missed it when making delivery configs required)

Reply:
Honestly I don't know why I added those. Removing.